apis/kueue/v1beta1/workload_types.go

```diff
@@ -287,6 +287,13 @@ const (
 
 	// WorkloadEvicted means that the Workload was evicted by a ClusterQueue
```

Comment:
Update comment to include that if the reason is Preempted, another condition accompanies it with more details.

Reply:
Done